citizenvoice/tests/test_models.py

```diff
@@ -0,0 +1,30 @@
+from django.test import TestCase
+from apiapp.models import Question, Survey, Answer, Response
+from django.contrib.auth.models import User
+from datetime import date
+
+
+class ModelTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        print("setUpTestData: Run once to set up non-modified data for all class methods.")
+        user = User(username='testuser', password='testpass')
+        user.save()
+        survey = Survey(name='Test Survey 1', description='This is used to test things',
+                        display_method=1, template='abcd', publish_date=date.today(),
+                        expire_date=date.today(), redirect_url='www.google.com', author=user)
```

Comment:
Rename 'author' to 'designer'

Reply:
@Gerdusvdl  will you push the changes as part of this pull request or will you plan to create a new one?

Reply:
Yes sorry, will push these changes in this PR before the end of the day

Reply:
@manuGil  Sorry, I'm still trying to fix the database issues, will push as soon as I've figured them out/ we make another plan.

Reply:
Done, with PostgreSQL workaround